src/main/scala/z3/scala/dsl/Trees.scala

```diff
@@ -93,6 +93,10 @@ case class IntConstant(value : Int) extends Tree[IntSort] {
   private[dsl] def build(z3 : Z3Context) = z3.mkInt(value, z3.mkIntSort)
 }
 
+case class CharConstant(value : Char) extends Tree[BVSort] {
+  private[dsl] def build(z3 : Z3Context) = z3.mkInt2BV(16, z3.mkInt(value.toInt, z3.mkIntSort()))
```

Comment:
You probably want to steer away from Int2BV as its performance is pretty terrible. In this case, it might actually be fast since we're giving it a constant and any decent preprocessor will notice this, but I would use mkNumeral just to be sure.